CAPSTONE PROJECT MODULE ONE

HanningtoneM opened the pull request:
- Used semantic HTML tags.
- Applied best practices in HTML code.
- Used CSS selectors correctly.
- Used CSS box model.
- Used Flexbox to place elements in the page.
- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
- Used GitHub Pages to deploy web pages.
- Applied JavaScript best practices and language style guides in code.
- Used JavaScript to manipulate DOM elements.
- Used JavaScript events.
- Used objects to store and access data.
- Communicated technical concepts to other technical people.
- Chose a topic that is different than the one in the original design.

- used a CSS framework (Bootstrap) for styling
- built these 2 pages: The home page. The about page.

- https://www.loom.com/share/d6c51b61b1f04562b520058f1ebf1a85 part one of the loom video
- https://www.loom.com/share/ec06a4d695094c7ab11ad55e876e88c8 part two of the loom video

HanningtoneM committed:
Add readme

HanningtoneM committed:
Add html and css file

HanningtoneM committed:
Add header and mobile menu

HanningtoneM committed:
Add Hero section both mobile and desktop

HanningtoneM committed:
Add program section and css

HanningtoneM committed:
Add footer and speakers and respective css

HanningtoneM committed:
Add about.html

HanningtoneM committed:
Add past events

HanningtoneM committed:
fix linter error

HanningtoneM committed:
Change the css file

HanningtoneM committed:
Change img in html

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Changes to image

HanningtoneM committed:
Add ./ on img

HanningtoneM committed:
Add ./ on img

HanningtoneM committed:
fix linters error

HanningtoneM committed:
Add ../ to background image

keroloussamy commented on about.html:
```diff
@@ -0,0 +1,243 @@
+        to enlighten society about the challenges facing women in the kenyan society 
+      </p>
+    </div>
+      <div class="contactText">
+        <p class="text-center">
+          feel free to reach out to us if you need further clarification about the 
+          work we do for the community <br>
+          <b class="contactEmail">hanningtonepompyy@gmail.com</b>
+        </p>
+      </div>
+</article>
+<article class="womenKenyaLogo">
+  <h2 class="text-center seconday-dark">Kenya Women’s Summit Logo</h2>
+  <div class="line"></div>
+  <div class="logoDescripiton">
+    <p class="text-center desktopDescription">
+      The Kenya women's summit brings together high-achieving women from all over the country. 
+      It gives focus to new ways of thinking and presents aspects we think we already understand from a new and unfamiliar perspective. 
+      This summit enables women to start a dialogue; women who shape their societies and their organisations – women who make their mark.
+      It directs attention to new markets and fields of action for companies and regions, enables the joint exploration of visions and strategies, 
+      and extends the boundaries of previous knowledge. 
+    </p>
```
- [x] Kindly the text here overlaps each other, and maybe it looks good on your screen, but you have also to be sure that your design works correctly on all widths.
- [x] Kindly you can give the paragraph some margin from left and right to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/168909793-63724db4-16ba-41dd-a396-33c2dbc677a3.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168909875-3654d80d-19a7-4264-98ab-04f59320138d.png)

keroloussamy commented on about.html:
```diff
@@ -0,0 +1,243 @@
+ </div>
+</div>
+</article>
+<article class="partners">
+  <h2 class="text-center secondary-light">partners</h2>
+  <div class="line"></div>
+  <div class="container d-flex partnerslogos">
+      <div class="partnerLogodiv">
+          <img src="./images/partner/google.svg" alt="google">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/mozilla.svg" alt="mozilla">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/microsoft.svg" alt="microsoft">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/zoom-communications.svg" alt="zoom">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/airbnb.svg" alt="airbnb">
+      </div>
+  </div>
+</article>
```
- [x] Kindly this section doesn't exist on the about page, it shows on the mobile version but on the desktop version, it does not exist.

Your design
![image](https://user-images.githubusercontent.com/35535130/168910175-ef266eac-568d-485e-89ec-94e8e8dd3c97.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168910265-2fada53d-87d6-48fa-a6ab-be33b1aed75a.png)

keroloussamy commented on about.html:
```diff
@@ -0,0 +1,243 @@
+      </div>
+  </div>
+</article>
+<footer>
+  <div class="container footerWrapper">
+      <div class="mt-5 footerLogoImage">
+        <a href="index.html"
+          ><img
+            src="./images/womenlogo3.png"
+            alt="women's logo"
+            class="footerWomenSummit"
+        /></a>
+      </div>
+      <div class="footerInfo">
+        <h4>@2022 Women's Summit.</h4>
+        <p class="mt-3">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Omnis,
+          suscipit veritatis!?
+        </p>
+      </div>
+    </div>
+  </footer>
```
- [x] Kindly change the background color for the footer section on the about page, to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/168910517-264602bb-80f1-4264-bd46-7a0bf13d3105.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168910466-7c14c284-7849-415e-8d63-582634910b57.png)

keroloussamy commented on README.md:
```diff
@@ -1,2 +1,46 @@
+- Github: [@HanningtoneM](https://github.com/HanningtoneM)
+
+
+## Credits
+
+Template from this [Figma](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Microverse](https://bit.ly/MicroverseTN)
+
```
- [x] Kindly you can mention the name of the author [Cindy Shin](https://www.behance.net/adagio07) instead of Microverse.

keroloussamy reviewed (changes requested):
Hi @HanningtoneM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HanningtoneM committed:
fix issues requested by the reviewer

HanningtoneM committed:
Add author

HanningtoneM commented:
fixed issues requested by the reviewer

- mentioned the name of the author [Cindy Shin](https://www.behance.net/adagio07) 
- I changed the background color for the footer section on the about page, to match the original design.
- I removed the partners section on the about page
- fixed text overlapping issue
- centered paragraph to appear like the original design

rustam-tolipov commented on README.md:
```diff
@@ -1,2 +1,46 @@
 # Women-summit
 women's summit to highlight some of the problems facing women
+
```
- [x] [OPTIONAL] Good job 👏, all requirements from the previous review are implemented ✔. I would suggest adding a screenshot for the website. Let's also add Twitter and LinkedIn account links in the **author** part 🙌

rustam-tolipov reviewed (approved):
## Hi @HanningtoneM 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

![Developer-512px-10](https://user-images.githubusercontent.com/69011963/151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6.gif)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

HanningtoneM committed:
Add bg color on the footer about page

HanningtoneM committed:
Add screenshot to readme
